spec/support/static_validation_helpers.rb

```diff
@@ -12,7 +13,8 @@
 module StaticValidationHelpers
   def errors
     target_schema = schema
-    validator = GraphQL::StaticValidation::Validator.new(schema: target_schema)
+    target_rules = defined?(rules) ? rules : GraphQL::StaticValidation::ALL_RULES
```

Comment:
The real ickiness comes from some validators assuming that previous ones will return `Visitor::SKIP`, otherwise they hit a Ruby error or start some infinite loop (eg https://github.com/rmosolgo/graphql-ruby/pull/330 as the most recent example)